packages/aik-route-config/hoistGlobalPlugin.ts

```diff
@@ -126,10 +126,8 @@ function makeVisitor(magicImport: string, currentModule: string): Visitor {
 			}
 		},
 		visitImportDeclaration(path) {
-			assert.ok(
-				this.state.phase === VisitorPhase.Initializing,
-				'Unexpected import declaration after initialization'
-			);
+			// Imports that come after the initialization are not relevant
```

Comment:
What does this mean actually? You mean here the import inside the script tag was previously flagged?

Reply:
Exactly that! The Astro file is split into multiple script files and this expectation is true only for one of them, failing in pre-processed script blocks.